Server/ReasnAPI/ReasnAPI.Tests/Services/CommentServiceTests.cs

```diff
@@ -0,0 +1,323 @@
+﻿using ReasnAPI.Models.Database;
+using ReasnAPI.Models.DTOs;
+using ReasnAPI.Services;
+using Moq;
+using Moq.EntityFrameworkCore;
+
+namespace ReasnAPI.Tests.Services {
+    [TestClass]
+    public class CommentServiceTests {
+        [TestMethod]
+        public void GetCommentById_CommentExist_CommentReturned() {
+            var mockContext = new Mock<ReasnContext>();
+
+            var event1 = new Event() {

```

Comment:
Do you really need to name it like event1? Additionally, remove redundant "()".

Reply:
Event is a keyword in C#